src/main/java/com/google/sps/api/authorization/AuthorizationRequester.java

```diff
@@ -82,7 +86,7 @@ public static String getRedirectUri(HttpServletRequest req) {
   public static GoogleAuthorizationCodeFlow newFlow() throws IOException {
     return new GoogleAuthorizationCodeFlow.Builder(HTTP_TRANSPORT, JSON_FACTORY,
         getClientCredential(), Arrays.asList(CalendarScopes.CALENDAR_EVENTS, TasksScopes.TASKS)).setDataStoreFactory(
-        DATA_STORE_FACTORY).setAccessType(ACCESS_TYPE).build();
+        DATA_STORE_FACTORY).setAccessType(ACCESS_TYPE).setApprovalPrompt(APPROVAL_PROMPT).addRefreshListener(new DataStoreCredentialRefreshListener(getUserId(), DATA_STORE_FACTORY)).build();
```

Comment:
It would be good to break this line in two.

Reply:
usually what works well is one method call per line. Also extract larger internal expressions into a separate variable. (i.e. the whole part with `Arrays.asList(CalendarScopes.CALENDAR_EVENTS, TasksScopes.TASKS))....`